ScriptureRenderingPipelineWorker/MergeHandler.cs

```diff
@@ -179,9 +179,11 @@ private void MergeWriterProject(ZipFileSystem projectZip, string? basePath, Merg
 	    {
 		    return;
 	    }
+	    
+	    var bookIndex = Utils.GetBookNumber(bookCode);
 				
 	    var usfm = renderer.Render(usfmObject);
-	    output.Add($"{bookCode}.usfm", usfm);
+	    output.Add($"{(bookIndex == 0 ? "" : $"{bookIndex}-")}{bookCode}.usfm", usfm);
```

Comment:
For compatibility with other tools, `bookCode` should be uppercase.

Reply:
Got it, changed.